### ISSUE TITLE
Demo?

### ISSUE THREAD

**Issue by di48l069** [TrackingTeam/Inventory-Track#37]
Hi,

Do you have a live demo of this?

**Comment by tankerkiller125** [comment]
I don't believe we do. However I would ask @worldmc since he's essentially the lead on this project.

**Comment by worldmc** [comment]
Hello @di48l069,

I do not have a demo running of this, however, i could set it up this weekend. 
If you want you can send me a email at [mailto:arjo@arjovelderman.eu](arjo@arjovelderman.eu) with a bit of information on what you are looking for, then i can maybe see if i can make a special version for you. 

Greetings, Arjo

**Comment by di48l069** [comment]
@worldmc Thank you I have mailed you

**Comment by di48l069** [comment]
Still no Demo..........

**Comment by worldmc** [comment]
True, work caught up to me and currently paid projects take priority. 

I hopefully will have time to do it this month.

**Comment by di48l069** [comment]
@worldmc Thanks I hope you find time to do so.
